core/modules/user/user.pages.inc

```diff
@@ -29,7 +29,38 @@ function user_autocomplete($string = '') {
  */
 function user_pass_page() {
   backdrop_set_title(t('Reset password'));
-  return backdrop_get_form('user_pass');
+  $user_register_form = backdrop_get_form('user_pass');
```

Comment:
Here and below, we should name this variable to avoid confusion:

```suggestion
  $user_pass_form = backdrop_get_form('user_pass');
```

Reply:
Renamed.